src/OrbitGl/MajorPagefaultTrack.h

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
```

Comment:
We cannot combine system memory track and cgroup memory track. But we can do this for the pagefault tracks. 😃 

I added a commit to remove the current `MajorPagefaultTrack` and `MinorPagefaultTrack` classes and create them with the `BasicPagefaultTrack`.

Reply:
First, Vicky is right about the system memory track and cgroup memory track, we cant merge them because of the template argument. But also, why would we merge them? 

I use Major and MinorPagefault tracks as an example: Why should this be merged to BasicPageFaultTrack?? I think some abstraction makes total sense here. There are quite a few differences between Major and Minor, mainly the tooltips. How I judge this: The decision is: One more (unnecessary) complex class vs. multiple simpler classes. And would say the second option is better. I would also argue, this is aligned with https://en.wikipedia.org/wiki/Single-responsibility_principle